packages/examples/packages/bip32/src/index.test.ts

```diff
@@ -141,6 +141,7 @@ describe('onRpcRequest', () => {
       });
 
       const ui = await response.getInterface();
```

Comment:
Now that I think about it, it seems easier to specify the expected type in `getInterface` rather than having to assert all the time.

```suggestion
      const ui = await response.getInterface('confirmation');
      // `ui` is now `ConfirmationDialog` type
```

I can take care of this in a separate PR.